packages/@react-stately/collections/docs/collections.mdx

```diff
@@ -296,7 +296,7 @@ function addPerson(name) {
 </ListBox>
 ```
 
-<!--
+{/*
```

Comment:
Why are these commeted out instead of deleted?

Reply:
it was a comment before, this is just the way that mdx comments need to be now, see https://github.com/adobe/react-spectrum/pull/2701#discussion_r772684179